crates/bevy_render/src/camera/camera.rs

```diff
@@ -86,8 +86,8 @@ pub struct ComputedCameraValues {
 pub struct Camera {
     /// If set, this camera will render to the given [`Viewport`] rectangle within the configured [`RenderTarget`].
     pub viewport: Option<Viewport>,
-    /// Cameras with a lower priority will be rendered before cameras with a higher priority.
-    pub priority: isize,
+    /// Cameras with a higher ordering are rendered later, and thus on top of lower ordering cameras.
```

Comment:
I like the rename, but wouldn't `order` be more grammatically correct? I am a non-native speaker, but `ordering` sounds wrong to me.

Reply:
Another word we could use is `sequence`. I am not sure which is better though.

Reply:
Both order and ordering seem acceptable to me here.

Order is a bit nicer though?

Reply:
I prefer `order` personally.